bin/server.js

```diff
@@ -8,9 +10,33 @@ var log         = require('../middleware/logger').child({
 });
 
 // set port for server
-var port = process.env.PORT || 3000;
+var port = process.env.PORT || 8999;
 
 // connect app to database
 mongoose.connect(mongoURI);
 
-app.listen(port);
+mongoose.connection.on('error', function(err){
+  log.error('error connecting to DB :%s', mongoURI, err);
+
+})
+
+mongoose.connection.on('open', function(err){
+  log.info('successfully connecting to DB :%s', mongoURI);
+  instantiateServer()
+})
+
+
+function instantiateServer(){
+
+  app.use(bodyParser.urlencoded({extended: true}));
+  app.use(bodyParser.json());
+
+  var router = express.Router();
```

Comment:
as a suggestion ... this should perhaps be defined at a global level also underneath the app declaration ... i just saw this and I've made the same mistake although it is not a major issue, just a pointer